### PR TITLE
tiup: refine expressions in playground.md

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

refine expressions in playground.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/pull/2480/

### CODE REVIEW

File: reference/tools/tiup/playground.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ playground 组件的基本用法：
 tiup playground [version] [flags]
 ```
 
-如果直接执行 `tiup playground` 命令，则会使用本地安装的 TiDB/TiKV/PD 或它们的稳定版本启动一个由 1 个 TiKV、1 个 TiDB、1 个 PD 实例构成的集群。该命令实际做了以下事情：
+如果直接执行 `tiup playground` 命令，则 TiUP playground 会使用本地安装的 TiDB/TiKV/PD 组件或者安装这些组件的稳定版本，来启动一个由 1 个 TiKV、1 个 TiDB、1 个 PD 实例构成的集群。该命令实际做了以下事情：
 
 - 因为没有指定版本，TiUP 会先查找已安装的 playground 的最新版本，假设已安装的最新版为 v0.0.6，则该命令相当于 tiup playground:v0.0.6
-- 如果 playground 组件从未安装过任何版本，TiUP 会先将其安装最新稳定版，然后再启动运行实例
-- 因为 playground 没有指定 TiDB/PD/TiKV 各组件的版本，默认情况下，它会使用各组件的最新 release 版本，假设当前为 v4.0.0-rc，则该命令相当于 tiup playground:v0.0.6 v4.0.0-rc
+- 如果 playground 从未安装过任何版本的 TiDB/TiKV/PD 组件，TiUP 会先将安装这些组件的最新稳定版，然后再启动运行这些组件的实例
```
Comment (yikeke) [inline reference/tools/tiup/playground.md:21]:
```suggestion
- 如果 playground 从未安装过任何版本的 TiDB/TiKV/PD 组件，TiUP 会先安装这些组件的最新稳定版，然后再启动运行这些组件的实例
```

File: reference/tools/tiup/playground.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ playground 组件的基本用法：
 tiup playground [version] [flags]
 ```
 
-如果直接执行 `tiup playground` 命令，则会使用本地安装的 TiDB/TiKV/PD 或它们的稳定版本启动一个由 1 个 TiKV、1 个 TiDB、1 个 PD 实例构成的集群。该命令实际做了以下事情：
+如果直接执行 `tiup playground` 命令，则 TiUP playground 会使用本地安装的 TiDB/TiKV/PD 组件或者安装这些组件的稳定版本，来启动一个由 1 个 TiKV、1 个 TiDB、1 个 PD 实例构成的集群。该命令实际做了以下事情：
 
 - 因为没有指定版本，TiUP 会先查找已安装的 playground 的最新版本，假设已安装的最新版为 v0.0.6，则该命令相当于 tiup playground:v0.0.6
```
Comment (yikeke) [inline reference/tools/tiup/playground.md:20]:
```suggestion
- 因为该命令没有指定 playground 的版本，TiUP 会先查找已安装的 playground 的最新版本，假设已安装的 playground 最新版为 v0.0.6，则该命令相当于 tiup playground:v0.0.6
```

File: reference/tools/tiup/playground.md [hunk 3]
```diff
@@ -15,13 +15,13 @@ playground 组件的基本用法：
...
-- 如果 playground 组件从未安装过任何版本，TiUP 会先将其安装最新稳定版，然后再启动运行实例
-- 因为 playground 没有指定 TiDB/PD/TiKV 各组件的版本，默认情况下，它会使用各组件的最新 release 版本，假设当前为 v4.0.0-rc，则该命令相当于 tiup playground:v0.0.6 v4.0.0-rc
+- 如果 playground 从未安装过任何版本的 TiDB/TiKV/PD 组件，TiUP 会先安装这些组件的最新稳定版，然后再启动运行这些组件的实例
+- 因为 playground 没有指定 TiDB/PD/TiKV 各组件的版本，默认情况下，它会使用各组件的最新发布版本，假设当前为 v4.0.0-rc，则该命令相当于 tiup playground:v0.0.6 v4.0.0-rc
```
Comment (yikeke) [inline reference/tools/tiup/playground.md:22]:
```suggestion
- 因为该命令没有指定 TiDB/PD/TiKV 各组件的版本，默认情况下，它会使用各组件的最新发布版本，假设当前为 v4.0.0-rc，则该命令相当于 tiup playground:v0.0.6 v4.0.0-rc
```

File: reference/tools/tiup/playground.md [hunk 4]
```diff
@@ -15,13 +15,13 @@ playground 组件的基本用法：
...
 - 因为 playground 也没有指定各组件的个数，默认情况下，它会启动由 1 个 TiDB、1 个 TiKV 和 1 个 PD 构成的最小化集群
```
Comment (yikeke) [inline reference/tools/tiup/playground.md:23]:
```suggestion
- 因为该命令也没有指定各组件的个数，默认情况下，它会启动由 1 个 TiDB、1 个 TiKV 和 1 个 PD 实例构成的最小化集群
```

File: reference/tools/tiup/playground.md [hunk 5]
```diff
@@ -15,13 +15,13 @@ playground 组件的基本用法：
...
-- 因为没有指定版本，TiUP 会先查找已安装的 playground 的最新版本，假设已安装的最新版为 v0.0.6，则该命令相当于 tiup playground:v0.0.6
-- 如果 playground 组件从未安装过任何版本，TiUP 会先将其安装最新稳定版，然后再启动运行实例
-- 因为 playground 没有指定 TiDB/PD/TiKV 各组件的版本，默认情况下，它会使用各组件的最新 release 版本，假设当前为 v4.0.0-rc，则该命令相当于 tiup playground:v0.0.6 v4.0.0-rc
-- 因为 playground 也没有指定各组件的个数，默认情况下，它会启动由 1 个 TiDB、1 个 TiKV 和 1 个 PD 构成的最小化集群
-- 在依次启动完各个组件后，playground 会提醒启动成功，并告诉你一些有用的信息，譬如如何通过 MySQL 客户端连接集群、如何访问 dashboard
+- 因为该命令没有指定 playground 的版本，TiUP 会先查找已安装的 playground 的最新版本，假设已安装的 playground 最新版为 v0.0.6，则该命令相当于 tiup playground:v0.0.6
+- 如果 playground 从未安装过任何版本的 TiDB/TiKV/PD 组件，TiUP 会先安装这些组件的最新稳定版，然后再启动运行这些组件的实例
+- 因为该命令没有指定 TiDB/PD/TiKV 各组件的版本，默认情况下，它会使用各组件的最新发布版本，假设当前为 v4.0.0-rc，则该命令相当于 tiup playground:v0.0.6 v4.0.0-rc
+- 因为该命令也没有指定各组件的个数，默认情况下，它会启动由 1 个 TiDB、1 个 TiKV 和 1 个 PD 实例构成的最小化集群
+- 在依次启动完各个组件后，playground 会提醒启动成功，并告诉你一些有用的信息，譬如如何通过 MySQL 客户端连接集群、如何访问 TiDB Dashboard 等
```
Comment (yikeke) [inline reference/tools/tiup/playground.md:24]:
```suggestion
- 在依次启动完各个 TiDB 组件后，playground 会提醒集群启动成功，并告诉你一些有用的信息，譬如如何通过 MySQL 客户端连接集群、如何访问 TiDB Dashboard 等
```

File: reference/tools/tiup/playground.md [hunk 6]
```diff
@@ -96,14 +96,14 @@ tiup playground --db.binpath /xx/tidb-server
 tiup playground v3.0.10 --db 3 --pd 3 --kv 3
 ```
 
-## 快速连接到启动的 playground 集群
+## 快速连接到由 playground 启动的 TiDB 集群
```
Comment (yikeke) [inline reference/tools/tiup/playground.md:99]: PTAL at this section. @lucklove

**Comment by yikeke** [comment]
PTAL @lucklove

File: reference/tools/tiup/playground.md (hunk 6 above)
Comment (lucklove) [reply reference/tools/tiup/playground.md:99]: LGTM

**Pull request merged**